### DISCUSSION TITLE
Suggestion: Populate the whole 'From Text' tab when you drop an image into the prompt box

### DISCUSSION THREAD

**Discussion by gruevy** [vladmandic/sdnext#972] [Ideas]
Currently, if you drop a .png file with generation info into the prompt field, it'll fill the prompt field with that info. But it puts the negative prompt, steps, sample, cfg scale, etc, in the prompt field too. It'd be sweet if it put all that stuff where it needs to go instead.

**Comment by vladmandic** [comment]
can you open a feature request? its a good suggestion, but here it will drop off my radar.

**Reply by gruevy** [reply]
Yep, not a frequent github user, didn't know that option was there. I'll do it now
